dbms/src/Common/ODBCBridgeHelper.cpp

```diff
@@ -36,13 +36,13 @@ ODBCBridgeHelper::ODBCBridgeHelper(
 void ODBCBridgeHelper::startODBCBridge() const
 {
     Poco::Path path{config.getString("application.dir", "")};
-    path.setFileName("clickhouse");
+    path.setFileName("clickhouse-odbc-bridge");
```

Comment:
Now table function odbc will not work if someone use standalone binary without symlinks.